KifuSF/UIAnimatedButton.swift

```diff
@@ -137,7 +123,7 @@ class KFButton: UIButton, UIConfigurable {
         case .pressed:
             buttonAnimator = nil
         default:
-            let animator = UIViewPropertyAnimator(duration: KFButton.animationDuration, curve: .linear, animations: { [unowned self] in
+            let animator = UIViewPropertyAnimator(duration: UIAnimatedButton.animationDuration, curve: .linear, animations: { [unowned self] in
```

Comment:
@Pondorasti be careful when you use `unowned` in your capture lists. In an animation block, `self` could be removed from memory before the animation ends